docs/products/flink/concepts/custom-jars.rst

```diff
@@ -0,0 +1,31 @@
+Custom JARs in Aiven for Apache Flink®
+=================================================
+
+Aiven for Apache Flink® allows you to upload, deploy, and manage your own Java code with custom JARs. This feature aims to expand the capabilities of Aiven for Apache Flink, allowing you to add custom functions that extend beyond the typical SQL-based features. With Custom JARs, you can swiftly develop and maximize the potential of your Aiven for Flink application.
```

Comment:
custom functions may distract people into thinking of user-defined functions, which are something different altogther and not yet offered. 

"allowing you to add custom capabilities that extend beyond the default SQL features"